From f0292008c017261421876fbff072bbd43f950b5d Mon Sep 17 00:00:00 2001
From: Jernej Skrabec <jernej.skrabec@gmail.com>
Date: Sun, 9 May 2021 11:52:43 +0200
Subject: [PATCH 382/478] ARM: dts: sun8i: r40: bananapi-m2-ultra: Enable HDMI
 audio

BananaPi M2 Ultra has HDMI output and thus supports HDMI audio. Enable
it.

Signed-off-by: Jernej Skrabec <jernej.skrabec@gmail.com>
---
 arch/arm/boot/dts/sun8i-r40-bananapi-m2-ultra.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/sun8i-r40-bananapi-m2-ultra.dts b/arch/arm/boot/dts/sun8i-r40-bananapi-m2-ultra.dts
index a6a1087a0c9b..87cf03574991 100644
--- a/arch/arm/boot/dts/sun8i-r40-bananapi-m2-ultra.dts
+++ b/arch/arm/boot/dts/sun8i-r40-bananapi-m2-ultra.dts
@@ -164,6 +164,10 @@ axp22x: pmic@34 {
 
 #include "axp22x.dtsi"
 
+&i2s2 {
+	status = "okay";
+};
+
 &ir0 {
 	status = "okay";
 };
@@ -299,6 +303,10 @@ &reg_eldo3 {
 	regulator-name = "vdd1v2-sata";
 };
 
+&sound_hdmi {
+	status = "okay";
+};
+
 &tcon_tv0 {
 	status = "okay";
 };
-- 
2.35.3

